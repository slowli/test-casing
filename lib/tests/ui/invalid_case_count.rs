use test_casing::test_casing;

#[test_casing("2", ["test", "this"])]
fn tested_function(_arg: &str) {
    // Does nothing
}

#[test_casing(0, [])]
fn other_tested_function(_arg: &str) {
    // Does nothing
}

fn main() {}
