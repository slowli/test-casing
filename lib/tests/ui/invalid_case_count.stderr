error: expected integer literal
 --> tests/ui/invalid_case_count.rs:3:15
  |
3 | #[test_casing("2", ["test", "this"])]
  |               ^^^

error: number of test cases must be positive
 --> tests/ui/invalid_case_count.rs:8:15
  |
8 | #[test_casing(0, [])]
  |               ^
